diff --git a/node_modules/ofetch/dist/shared/ofetch.37386b05.mjs b/node_modules/ofetch/dist/shared/ofetch.37386b05.mjs
index cae0a55..b611e38 100644
--- a/node_modules/ofetch/dist/shared/ofetch.37386b05.mjs
+++ b/node_modules/ofetch/dist/shared/ofetch.37386b05.mjs
@@ -234,7 +234,14 @@ function createFetch(globalOptions = {}) {
         clearTimeout(abortTimeout);
       }
     }
-    const hasBody = context.response.body && !nullBodyResponses.has(context.response.status) && context.options.method !== "HEAD";
+    const hasBody =
+      (context.response.body ||
+        // React Native whatwg-fetch check. Can be removed when the following occurs
+        // 1) https://github.com/JakeChampion/fetch/issues/1454 is fixed
+        // 2) React Native upgrades to the whatwg-fetch version that has this fix
+        (context.response)._bodyInit) &&
+      !nullBodyResponses.has(context.response.status) &&
+      context.options.method !== "HEAD";
     if (hasBody) {
       const responseType = (context.options.parseResponse ? "json" : context.options.responseType) || detectResponseType(context.response.headers.get("content-type") || "");
       switch (responseType) {
diff --git a/node_modules/ofetch/dist/shared/ofetch.897a6909.cjs b/node_modules/ofetch/dist/shared/ofetch.897a6909.cjs
index 099990b..4e548bc 100644
--- a/node_modules/ofetch/dist/shared/ofetch.897a6909.cjs
+++ b/node_modules/ofetch/dist/shared/ofetch.897a6909.cjs
@@ -240,7 +240,14 @@ function createFetch(globalOptions = {}) {
         clearTimeout(abortTimeout);
       }
     }
-    const hasBody = context.response.body && !nullBodyResponses.has(context.response.status) && context.options.method !== "HEAD";
+    const hasBody =
+      (context.response.body ||
+        // React Native whatwg-fetch check. Can be removed when the following occurs
+        // 1) https://github.com/JakeChampion/fetch/issues/1454 is fixed
+        // 2) React Native upgrades to the whatwg-fetch version that has this fix
+        (context.response)._bodyInit) &&
+      !nullBodyResponses.has(context.response.status) &&
+      context.options.method !== "HEAD";
     if (hasBody) {
       const responseType = (context.options.parseResponse ? "json" : context.options.responseType) || detectResponseType(context.response.headers.get("content-type") || "");
       switch (responseType) {
