diff --git a/dist/shared/ofetch.37386b05.mjs b/dist/shared/ofetch.37386b05.mjs
index cae0a5501b3b9c681509371708129a4ffbeb1166..b611e38de74dd0a5865801bbd67dc32534addbcf 100644
--- a/dist/shared/ofetch.37386b05.mjs
+++ b/dist/shared/ofetch.37386b05.mjs
@@ -234,7 +234,14 @@ function createFetch(globalOptions = {}) {
         clearTimeout(abortTimeout);
       }
     }
-    const hasBody = context.response.body && !nullBodyResponses.has(context.response.status) && context.options.method !== "HEAD";
+    const hasBody =
+      (context.response.body ||
+        // React Native whatwg-fetch check. Can be removed when the following occurs
+        // 1) https://github.com/JakeChampion/fetch/issues/1454 is fixed
+        // 2) React Native upgrades to the whatwg-fetch version that has this fix
+        (context.response)._bodyInit) &&
+      !nullBodyResponses.has(context.response.status) &&
+      context.options.method !== "HEAD";
     if (hasBody) {
       const responseType = (context.options.parseResponse ? "json" : context.options.responseType) || detectResponseType(context.response.headers.get("content-type") || "");
       switch (responseType) {
diff --git a/dist/shared/ofetch.897a6909.cjs b/dist/shared/ofetch.897a6909.cjs
index 099990b5b3d70e9b3c191a58d0bc96988ad8d515..4e548bcd2d267f5deb149f3617a42f9707dd030b 100644
--- a/dist/shared/ofetch.897a6909.cjs
+++ b/dist/shared/ofetch.897a6909.cjs
@@ -240,7 +240,14 @@ function createFetch(globalOptions = {}) {
         clearTimeout(abortTimeout);
       }
     }
-    const hasBody = context.response.body && !nullBodyResponses.has(context.response.status) && context.options.method !== "HEAD";
+    const hasBody =
+      (context.response.body ||
+        // React Native whatwg-fetch check. Can be removed when the following occurs
+        // 1) https://github.com/JakeChampion/fetch/issues/1454 is fixed
+        // 2) React Native upgrades to the whatwg-fetch version that has this fix
+        (context.response)._bodyInit) &&
+      !nullBodyResponses.has(context.response.status) &&
+      context.options.method !== "HEAD";
     if (hasBody) {
       const responseType = (context.options.parseResponse ? "json" : context.options.responseType) || detectResponseType(context.response.headers.get("content-type") || "");
       switch (responseType) {
